<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimManagerDialog</class>
 <widget class="QDialog" name="AnimManagerDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animation Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,90,5">
     <item>
      <widget class="QComboBox" name="cbAnims">
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="dirTabs">
       <property name="currentIndex">
        <number>7</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>North</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="AnimationSetupWidget" name="northAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>North East</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="AnimationSetupWidget" name="northeastAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>East</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="AnimationSetupWidget" name="eastAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>South East</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="AnimationSetupWidget" name="southeastAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>South</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="AnimationSetupWidget" name="southAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>South West</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="AnimationSetupWidget" name="southwestAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>West</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="AnimationSetupWidget" name="westAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>North West</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="AnimationSetupWidget" name="northwestAnim" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,30,15,15,15,15">
       <item>
        <widget class="QPushButton" name="bDone">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bDeleteAnim">
         <property name="text">
          <string>Delete Animation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bAddAnim">
         <property name="text">
          <string>Add New Animation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bImportAnim">
         <property name="text">
          <string>Import Animation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bExportAnim">
         <property name="text">
          <string>Export Animation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimationSetupWidget</class>
   <extends>QWidget</extends>
   <header>AnimationSetupWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
